Dockerfile

```diff
@@ -24,6 +24,10 @@ COPY --from=builder /lib64/ld-linux-x86-64.so.2 /lib64/ld-linux-x86-64.so.2
 COPY --from=usermanager /etc/ssl/certs /etc/ssl/certs
 COPY --from=usermanager /etc/passwd /etc/passwd
 COPY --from=usermanager /etc/group /etc/group
+RUN mkdir -p /home/mx/config
+RUN mkdir -p /home/mx/logs
+RUN chown ${USERNAME} /home/mx/config
+RUN chown ${USERNAME} /home/mx/logs
```

Comment:
Can also be done in a single `run` step (thus less Docker "layers"), but it's also perfect as it is (since those individual layers are minimal).